README.md

```diff
@@ -9,7 +9,7 @@ This action enables you to fetch Doppler secrets for use in your GitHub Actions.
 The action can be configured in two ways:
 
 * Service Token (recommended)
```

Comment:
Can we make OIDC the `(recommended)` method now?

Reply:
👍 I tweaked a bit to call out that you need the Team or Enterprise plan for service accounts. Plus reordered the README a bit.


e14ee9b:
> Add OIDC to README


47ddb19:
> Move Service Token README section below Service Accounts